modules/rtbhouseBidAdapter.js

```diff
@@ -8,67 +8,34 @@ const REGIONS = ['prebid-eu', 'prebid-us', 'prebid-asia'];
 const ENDPOINT_URL = 'creativecdn.com/bidder/prebid/bids';
 const DEFAULT_CURRENCY_ARR = ['USD']; // NOTE - USD is the only supported currency right now; Hardcoded for bids
 
-/**
- * Helpers
- */
-
-function buildEndpointUrl(region) {
-  return 'https://' + region + '.' + ENDPOINT_URL;
-}
-
-/**
- * Produces an OpenRTBImpression from a slot config.
- */
-function mapImpression(slot) {
-  return {
-    id: slot.bidId,
-    banner: mapBanner(slot),
-    tagid: slot.adUnitCode.toString(),
-  };
-}
-
-/**
- * Produces an OpenRTB Banner object for the slot given.
- */
-function mapBanner(slot) {
-  return {
-    w: slot.sizes[0][0],
-    h: slot.sizes[0][1],
-    format: mapSizes(slot.sizes)
-  };
-}
-
-/**
- * Produce openRTB banner.format object
- */
-function mapSizes(slotSizes) {
-  const format = [];
-  slotSizes.forEach(elem => {
-    format.push({
-      w: elem[0],
-      h: elem[1]
-    });
-  });
-  return format;
-}
-
-/**
- * Produces an OpenRTB site object.
- */
-function mapSite(validRequest) {
-  const pubId = validRequest && validRequest.length > 0 ? validRequest[0].params.publisherId : 'unknown';
-  return {
-    publisher: {
-      id: pubId.toString(),
+// Codes defined by OpenRTB Native Ads 1.1 specification
+export const OPENRTB = {
+  NATIVE: {
+    IMAGE_TYPE: {
+      ICON: 1,
+      MAIN: 3,
+    },
+    ASSET_TYPE: {
+      TITLE: 1,
```

Comment:
Asset does not have type prop, as far as I know. The naming convention is probably misleading

Reply:
Fixed.